src/components/MessagesList/MessagesGroup/Message/Quote/Quote.vue

```diff
@@ -0,0 +1,135 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<docs>
+This component is intended to be used both in `NewMessageForm` and `Message`
+components.
+</docs>
+
+<template>
```

Comment:
So, you finally gave up having one single component for all messages?
What issues did you encountered?

Reply:
I will eventually do it :)

Reply:
Eventually do what?

Reply:
Make quote and regular message the same component :)

Reply:
Wanna see it now?? 
..OK :sweat_smile: